pkgs/tools/X11/xpointerbarrier/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, xorg, fetchurl }:
+stdenv.mkDerivation rec {
+  name = "xpointerbarrier-${version}";
+  version = "17.10";
+
+  src = fetchurl {
```

Comment:
Can we use fetchFromGitHub here instead?

Reply:
Sure. I've amended commit.